sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -63,6 +102,24 @@ export interface OnMessage {
     (message: ServiceBusMessage): Promise<void>;
 }
 
+export { ProxySettings }
+
+// @public
+export interface Queue extends QueueOptions {
+    _?: any;
```

Comment:
What is `_` about here?

Reply:
Added in documentation about it, this was being returned by the service and we were previously exposing this to the user. Hence, included it in.

Reply:
We have the _response the has the entire content, so we dont need this